src/components/NewPost/ImageCaption/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.css';
+import TextField from '@material-ui/core/TextField';
+
+const ImageCaption = () => (
```

Comment:
this is a bad name for this component, i imagine a component that's ImageCaption to only hold text